docs/configuration.asciidoc

```diff
@@ -73,7 +73,20 @@ The URL to where the APM Server is deployed.
 
 By default the agent will validate the TLS/SSL certificate of the APM Server if using HTTPS.
 You can switch this behavior off by setting this option to `false`.
-Disabling validation is normally required if using self-signed certificates.
+
+[[server-ca-cert]]
+==== `serverCaCert`
+
+* *Type:* String
+* *Env:* `ELASTIC_APM_SERVER_CA_CERT`
+
+By default the agent will validate the TLS/SSL certificate of the APM Server using the
```

Comment:
can we mention somewhere as `path to the cert location` or `file path`?  Just by going over the docs, I was confused with whether user had to read the file and pass the contents.

Reply:
If we want users to only be able to provide a file-path and not the content of the certificate, we might consider renaming the config option to `serverCaCertFile` to be more explicit. This would also allow us to add another config option in the future called simply `serverCaCert` which you could use to provide the actual cert body (a feature which I would find quite useful in certain scenarios as you might get the cert body from a configuration system or an environment variable and not actually have it on disk). This could of course also be achieved in the same config option by detecting the cert `BEGIN` signature.

Reply:
Yeah, I was on the fence with this. My instinct was to call it `serverCaCertFile`, but then there's the existing Ruby agent config. I'll open an issue on the Ruby agent to deprecate/change its config name.

Reply:
Opened https://github.com/elastic/apm-agent-ruby/issues/650, and I've renamed `serverCaCert` to `serverCaCertFile`.

@vigneshshanmugam the docs already said

> You can set this option to the path of a file containing a CA certificate that will be used instead.

But I guess part of the confusion came from the attribute name, so hopefully your concern is also addressed. Let me know if not.

Reply:
Ahh, i missed the docs part. I just glimpsed over and was wondering about wide vareity of options that we can pass here like string, string[] or buffer. But yeah the config option change clearly conveys it. Thanks 👍